pkg/tsdb/cloudmonitoring/time_series_filter.go

```diff
@@ -193,6 +193,9 @@ func (timeSeriesFilter *cloudMonitoringTimeSeriesFilter) parseResponse(queryRes
 							valueField,
 						},
 						RefID: timeSeriesFilter.RefID,
+						Meta: &data.FrameMeta{
+							ExecutedQueryString: executedQueryString,
```

Comment:
Would you add a test or add to an existing test the use of this field `ExecutedQueryString`?

Reply:
I add assert for this field.